qa/L0_server_launch_modes/test.sh

```diff
@@ -24,24 +24,31 @@ REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
 MODEL_REPOSITORY=${MODEL_REPOSITORY:="/mnt/dldata/inferenceserver/$REPO_VERSION/libtorch_model_store"}
 CHECKPOINT_REPOSITORY=${CHECKPOINT_REPOSITORY:="/mnt/dldata/inferenceserver/model_analyzer_checkpoints"}
 MODEL_NAMES="vgg19_libtorch"
-BATCH_SIZES="4"
-CONCURRENCY="4"
+TRITON_LAUNCH_MODES="local docker remote"
```

Comment:
Just wondering what were the issues with the Docker launch mode.

Reply:
There were instances where model-analyzer would be killed but triton server remains running. If there was an exception thrown by the docker API, there would be a stopped (but not removed) container named tritonserver.